frontend/src/api/usePutData.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Custom hook updating data.
+ * @param {string} url - adress of request
+ * @param {state} requestBody - body od request made by user
+ * @returns { string object bool }
```

Comment:
Let's expect this hook to return Object in the data. 
Error should be a string (we'll be using this for popup.
isLoading as far as I know the name of this type is boolean :D